### DISCUSSION TITLE
How to cancel free trial and use Pro features instantly?

### DISCUSSION THREAD

**Discussion by syncros-juozas** [vercel/community#2798] [Help] [answered]
### Summary

I have free trial but I want to cancel it and instantly use Pro that I would have. Right now it says that I would be downgraded to Hobby, but I don't want that

### Steps to Reproduce

Go to billing > "Cancel trial"

**Comment by smaeda-ks** [comment, marked as answer]
@syncros-juozas I think if you add a valid payment method (credit card) and hit the "cancel trial" button, it will actually switch to a non-trial Pro plan.

**Comment by andnig** [comment]
@syncros-juozas Did this work for you?
If I cancel the trial, I just get the info, that it will be degraded to a Hobby Plan at the end of the trial period. I also have a valid payment method set. 
If I click "cancel trial" I'm still in the trial (as the trial period is still ongoing) but I get a banner at the top informing me, that I will be downgraded to "hobby" in some days.
Do you have any tips for me?

![image](https://github.com/vercel/community/assets/30227237/9ec04e01-d244-48c2-a944-0702f44d59e5)

**Reply by hypnocill** [reply]
hello @andnig How did you proceed? Did you find a way to immediately stop the pro trial and immediately start the Pro plan?

**Reply by andnig** [reply]
@hypnocill no. I contacted vercel support and they set the remaining trial period to 24 hours, meaning I was upgraded a day after that, which was kinda sufficient.
According to support, they are working on a self-service way to do that, but not released yet.

If it's about postgres and the usage limits, a workaround would be to create a new team with PRO plan (without trial) and transfer the potgres database there. This happens immediately and without service interruption. However is obviously quite a hack...

**Reply by hypnocill** [reply]
I see, thank you very much! I guess I won't use the trial then and just go for the pro plan
